### PR TITLE
feat: Show background image in header

### LINKED ISSUE #358: Show event background in header view
Show event background and logo in header view

### PR DESCRIPTION
**Pull request by iamareebjamal**
Fixes #358 

- Switch to Picasso 2.6.0-SNAPSHOT
- Enable Picasso logging
- Fix first time image load issue
- Change bg color of nav header
- Change main activity to binding
- Add binding adapter for image loading
- Add cancel tag for picasso

screenshot-1501080559879

### CODE REVIEW

File: app/src/main/res/layout/main_nav_header.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <variable
+            name="event"
+            type="org.fossasia.openevent.app.common.data.models.Event" />
+    </data>
+
+    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/nav_header_height">
+
+        <ImageView
```
Comment (mejariamol) [inline app/src/main/res/layout/main_nav_header.xml]: set `android:scaleType` to `centerCrop` or `fitCenter` to ensure an image always fits to the header.

**Comment by iamareebjamal** [comment]
@mejariamol Please review

File: app/src/main/res/layout/main_nav_header.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        <variable
+            name="event"
+            type="org.fossasia.openevent.app.common.data.models.Event" />
+    </data>
+
+    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/nav_header_height"
+        android:background="@color/grey_500">
```
Comment (mejariamol) [inline app/src/main/res/layout/main_nav_header.xml]: please change this color. 
1. when no image, it looks really dull and incomplete
2. also nav item selection color shade is similar so not looking good.

Comment (iamareebjamal) [inline app/src/main/res/layout/main_nav_header.xml]: Which color should be used then?

Comment (mejariamol) [inline app/src/main/res/layout/main_nav_header.xml]: default image would be better

Comment (iamareebjamal) [inline app/src/main/res/layout/main_nav_header.xml]: Which default image?

Comment (mejariamol) [inline app/src/main/res/layout/main_nav_header.xml]: the one used on front end server or the front end banner.

Comment (mejariamol) [inline app/src/main/res/layout/main_nav_header.xml]: like this dark background

bg

**Comment by iamareebjamal** [comment]
@mejariamol Please review. This was updated hours ago

**Pull request merged**